exampleSite/config.toml

```diff
@@ -37,6 +37,10 @@ copyright = "© {year}"
   #   "Twitter", "GitHub", "Email", "Facebook", "GitLab", "Instagram", "LinkedIn", "YouTube"
   iconTitles = ["Twitter", "GitHub"]
 
+  [params.blog]
+    # Enables summary view on blogs
+    summary = true
```

Comment:
I think a verb `showSummary` is better (more informative and meaningful) than just `summary`.

Reply:
```
[params]
  [param.blog]
    summary = true

```
is updated as 

```
[params]
  showSummary = true
```